tardis/plasma/properties/partition_function.py

```diff
@@ -275,3 +301,20 @@ class PartitionFunction(ProcessingPlasmaProperty):
     def calculate(self, level_boltzmann_factor):
         return level_boltzmann_factor.groupby(
             level=['atomic_number', 'ion_number']).sum()
+
+
+class ThermalLTEPartitionFunction(PartitionFunction):
```

Comment:
I'm not quite clear why we need to have them called `ThermalLTE` - as `LTE` means Local Thermal Equilibrium